x-pack/plugins/security_solution_serverless/public/get_started/card_step/index.tsx

```diff
@@ -93,108 +146,119 @@ const CardStepComponent: React.FC<{
       borderRadius="none"
       paddingSize="none"
       className={panelClassNames}
+      id={stepId}
       css={css`
-        padding: ${euiTheme.size.base};
-        margin: 0 ${euiTheme.size.s} 0;
+        overflow: hidden;
+        height: ${isExpandedStep ? expandedStepPanelHeight : euiTheme.size.xxxl};
+        transition: height ${HEIGHT_ANIMATION_DURATION}ms ease-out;
 
-        &.step-panel-collapsed:hover {
-          ${backgroundColorStyles.primary};
-          border-radius: ${euiTheme.border.radius.medium};
+        &.step-panel-expanded {
+          height: ${stepContentRef.current?.offsetHeight ? expandedStepPanelHeight : 'auto'};
+          transition: height ${HEIGHT_ANIMATION_DURATION}ms ease-in;
+        }
+
+        &.step-panel-collapsed,
+        &.step-panel-expanded.step-panel-collapsed {
+          height: ${euiTheme.size.xxxl};
```

Comment:
Same, can we extract all this CSS outside the component implementation? We can set only one to CSS for the panel parent, using classes for the children elements (`.step-icon { ... }`).